modules/core/src/main/resources/META-INF/classnames.properties

```diff
@@ -403,8 +403,6 @@ org.apache.ignite.internal.events.DiscoveryCustomEvent
 org.apache.ignite.internal.executor.GridExecutorService
 org.apache.ignite.internal.executor.GridExecutorService$1
 org.apache.ignite.internal.executor.GridExecutorService$TaskTerminateListener
-org.apache.ignite.internal.jdbc.JdbcConnectionValidationTask
-org.apache.ignite.internal.jdbc.JdbcConnectionValidationTask$1
```

Comment:
`org.apache.ignite.internal.processors.cache.query.jdbc.GridCacheQueryJdbcMetadataTask`
`org.apache.ignite.internal.processors.cache.query.jdbc.GridCacheQueryJdbcMetadataTask$JdbcDriverMetadataJob`
`org.apache.ignite.internal.processors.cache.query.jdbc.GridCacheQueryJdbcTask`
`org.apache.ignite.internal.processors.cache.query.jdbc.GridCacheQueryJdbcTask$JdbcDriverJob`
`org.apache.ignite.internal.processors.cache.query.jdbc.GridCacheQueryJdbcTask$JdbcDriverJob$1`
`org.apache.ignite.internal.processors.cache.query.jdbc.GridCacheQueryJdbcValidationTask`

Should be removed as well.